config/initializers/feature_decisions.rb

```diff
@@ -61,3 +61,8 @@
 
 OpenProject::FeatureDecisions.add :stages_and_gates,
                                   description: "Enables the under construction feature of stages and gates."
+
+OpenProject::FeatureDecisions.add :oidc_token_exchange,
```

Comment:
I just realized that some feature flags are defined on the scope of modules. Though I wouldn't know where to put this, since this flag will be crossing boundaries between `storages` and `openid_connect`.

Opinions?

Reply:
I could see an argument for both in the module or in global space. Would be useful to know of a previous occurrence of such problem :sweat_smile: 

But I think it's okay to keep it here since it's already written.

Reply:
I think following a recent discussion on Matrix, I'd now go for **this** location, because the feature is crossing boundaries and apparently putting all things into modules isn't of utmost importance.